test/util.ts

```diff
@@ -137,3 +137,15 @@ const bufferSerializer: jest.SnapshotSerializerPlugin = {
 export function addBufferSerializer(): void {
   expect.addSnapshotSerializer(bufferSerializer);
 }
+
+export function loadLocalFixtureDirectory(
+  fixturePath: string,
+  fixtureRoot = '.'
+): void {
+  const fixtureAbsPath = getFixturePath(fixturePath, fixtureRoot);
+
+  const config = mocked(_config);
```

Comment:
Do not mock admin config. Simply use setAdminConfig like in other tests and mock `util/fs` instead.

I think for this tests you do not need any mock at all.

Reply:
changed it by b2b5648ff3802be2446a3cc1a9ab3b6fde2f8390

You were right, I was able to remove all mocks